releasing/README.md

```diff
@@ -376,6 +376,42 @@ project [k8s-staging-kustomize].
 Commit and push your changes. Then create a PR to [k8s.io] to promote
 new images. Assign the PR to @monopole and @Shell32-natsu.
 
+## Update kustomize-in-kubectl
+
+[kubernetes/kubernetes]: https://github.com/kubernetes/kubernetes
```

Comment:
ln381 is omitted. Do you mean 

```bash
[kubernetes/kubernetes](https://github.com/kubernetes/kubernetes)
```

Reply:
It is not supposed to show up, but putting this here results in the text [kubernetes/kubernetes] in line 385 to follow this link. Similar style is above, in lines 363-364.

Reply:
Gotcha.